src/share/poudriere/jail.sh

```diff
@@ -198,6 +198,16 @@ update_jail() {
 	msg "Upgrading using ${METHOD}"
 	case ${METHOD} in
 	ftp|http|ftp-archive)
+		# In case we use FreeBSD dists and TORELEASE is present, check if it's a release branch.
+		if [ ! -z ${TORELEASE} ]; then
+		  case ${TORELEASE} in
+		    *-ALPHA*|*-CURRENT|*-PRERELEASE|*-STABLE)
+			msg_error "Only release branches are supported by freebsd-update(8)."
```

Comment:
The use of _freebsd-update(8)_ by poudriere is an implementation detail that is not documented and may change in the future; the user only knows they are using `-m ftp` (or _http_/_ftp-archive_) here, so we should not reference _freebsd-update(8)_.
Please change _freebsd-update(8)_ here to reference _${METHOD}_ such as `Only release branches are supported with method ${METHOD}"`